test/lib/constable/permalink_type_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule Constable.PermalinkTypeTest do
+  use ExUnit.Case, async: true
+
+  alias Constable.PermalinkType
+
+  test "type/0 is :id" do
+    assert :id == PermalinkType.type()
+  end
+
+  describe ".cast/1" do
+    test "casts integers" do
+      value = 1
+
+      casted_value = PermalinkType.cast(value)
```

Comment:
This could just be `cast_value`.

Reply:
Thanks! `cast_value` is much better. It made me laugh to think of `casted_value` once you pointed it out. Updated in 7ae90b94221df17c57b8918a8e0ffc19e8dd65e0